src/pages/Projects.tsx

```diff
@@ -59,10 +61,16 @@ const Projects = (): JSX.Element => {
     }
   };
 
+  const handleCreateProject = () => {
+    dispatch(clearCreatorState());
+    navigate(ROUTES.CREATOR);
+  };
+
   return (
     <div className="projects-page">
       <h2>Your latest projects</h2>
       {message}
+      <Button onClick={handleCreateProject}>Create new project</Button>

```

Comment:
New project should be created when clicking on the `Editor` in the navigation tab

Reply:
[AS-114](https://ar-chitect.atlassian.net/browse/AS-114)